### DISCUSSION TITLE
Asset tree get root failed

### DISCUSSION THREAD

**Discussion by werexa** [pimcore/pimcore#16492] [Q&A / Support]
I have pimcore project on local pc with docker for windows and WSL. 
I have a problem after downloaded project from company repo, I didn't see trees of object and asset on admin panel. 

URL: /admin/asset/tree-get-root

Message: Call to a member function isAllowed() on null

Trace:

in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/ElementControllerBase.php:72

#0 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Asset/AssetController.php(79): Pimcore\Bundle\AdminBundle\Controller\Admin\ElementControllerBase->treeGetRootAction(Object(Symfony\Component\HttpFoundation\Request))

#1 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(153): Pimcore\Bundle\AdminBundle\Controller\Admin\Asset\AssetController->treeGetRootAction(Object(Symfony\Component\HttpFoundation\Request))

#2 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)

#3 /var/www/html/vendor/symfony/http-kernel/Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)

#4 /var/www/html/public/index.php(36): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
![image001](https://github.com/pimcore/pimcore/assets/54885526/f0fc3ebc-93f9-4a7f-b0e1-851bcdba4598)

**Comment by faiyazalam** [comment]
@werexa  just clear the cache and it should work.
```
bin/console cache:clear
bin/console pimcore:cache:clear
```
